test/integration/buffer_accounting_integration_test.cc

```diff
@@ -810,6 +810,13 @@ TEST_P(Http2OverloadManagerIntegrationTest,
 }
 
 TEST_P(Http2OverloadManagerIntegrationTest, CanResetStreamIfEnvoyLevelStreamEnded) {
+  // This test is not applicable if expand_agnostic_stream_lifetime is enabled
```

Comment:
What will happen when the `expand_agnostic_stream_lifetime` is deprecated. Will the test have to be removed?

Reply:
Yes since this case won't occur anymore. The other tests of resetting streams will cover the reset path.